src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -560,12 +569,29 @@ private String findFakeCalendarById(String servicePointId, MultiMap queries) {
       String permanentLocation = holding.getString(ItemProperties.PERMANENT_LOCATION_ID);
       String temporaryLocation = holding.getString(ItemProperties.TEMPORARY_LOCATION_ID);
 
-      return item.put(ItemProperties.EFFECTIVE_LOCATION_ID,
+      return newItem.put(ItemProperties.EFFECTIVE_LOCATION_ID,
         ObjectUtils.firstNonNull(temporaryLocation, permanentLocation)
       );
     });
   }
 
+  private CompletableFuture<JsonObject> setItemStatusDateForItem(JsonObject oldItem,
+                                                                 JsonObject newItem) {
+    if (Objects.nonNull(oldItem)) {
+      JsonObject oldItemStatus = oldItem.getJsonObject(ItemProperties.STATUS_PROPERTY);
+      JsonObject newItemStatus = newItem.getJsonObject(ItemProperties.STATUS_PROPERTY);
+      if (ObjectUtils.allNotNull(oldItemStatus, newItemStatus)) {
+        if (!Objects.equals(oldItemStatus.getString("name"),
+          newItemStatus.getString("name"))) {
+          DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+          df.setTimeZone(TimeZone.getTimeZone("UTC"));
+          write(newItemStatus, "date", df.format(new Date()));
```

Comment:
I believe the JSON property writer should format the date time property appropriately, and the format isn’t needed.

Reply:
Finally, I catch your point. Thank you!